src/main/java/org/yinwang/pysonar/visitor/TypeInferencer.java

```diff
@@ -1098,7 +1098,7 @@ public Type apply(@NotNull FunType func,
                 }
             } else {
                 // usual method
-                if (selfType != null) {
+                if (selfType != null && !(selfType instanceof ModuleType)) {
```

Comment:
This is not the right place to check though. selfType should not be set at all here for package name. Need to find somewhere earlier than this.